Cesium3DTilesSelection/src/RasterOverlayUpsampler.cpp

```diff
@@ -72,6 +72,8 @@ RasterOverlayUpsampler::loadTileContent(const TileLoadInput& loadInput) {
         auto model = upsampleGltfForRasterOverlays(
             parentModel,
             TileID,
+            false,
+            "_CESIUMOVERLAY_",
```

Comment:
Maintainability comment, I'm seeing a lot of hits (25) of the `_CESIUMOVERLAY_` string literal in the code.

Any chance we can move this to a central place (like `RasterOverlayUtilities.h`), or reduce the number of occurrences?

Reply:
Ok, I added a constant to RasterOverlayUtilities as you suggested and replaced _most_ of the occurrences of `_CESIUMOVERLAY_` with it. There are some remaining in documentation, as well as remaining occurrences of `"_CESIUMOVERLAY_0"` in the tests because I thought a bit of repitition was better than something like `fmt::format("{}_{}", RasterOverlayUtilities::DEFAULT_TEXTURE_COORDINATE_BASE_NAME, 0)`.

Reply:
That's fair. And I suspected tedious code might be the result of that suggestion, so thank you for being sensible.